routes/check-if-project-exists.js

```diff
@@ -0,0 +1,81 @@
+var request = require("request");
+
+module.exports = function(config) {
+  return function(req, res) {
+    var publishURL = config.publishURL;
+    var token = config.cryptr.decrypt(req.session.token);
```

Comment:
As long as the user middleware runs on these routes, you don't need to decrypt the token:

https://github.com/mozilla/thimble.webmaker.org/blob/bramble/lib/middleware.js#L530-L533